### PR TITLE
Release notes to support 0.16 milestone 1

### PR DESCRIPTION
**Pull request by SueChaplain**
Draft release notes to support milestone 1.

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by pshipton** [comment]
Not sure about the timing of getting it merged, but fyi https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/325 will fix the javadoc issue for Java 8.

**Comment by pshipton** [comment]
lgtm
@DanHeidinga @andrewcraik @gita-omr @fjeremic any concerns?

**Comment by SueChaplain** [comment]
> Not sure about the timing of getting it merged, but fyi ibmruntimes/openj9-openjdk-jdk8#325 will fix the javadoc issue for Java 8.

Simon mentioned that he'd been working on this. Maybe we should leave this to 0.17 quarterly update when there will be builds at Adopt for JDK 8 (I'm assuming the JDK has the API doc in the package?)

**Comment by gita-omr** [comment]
LGTM

**Comment by pshipton** [comment]
> Maybe we should leave this to 0.17 quarterly update when there will be builds at Adopt for JDK 8

Right, good point.

**Comment by andrewcraik** [comment]
No concerns from me

File: doc/release-notes/0.16/0.16.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/6271">#6271</a></td>
+<td valign="top">Class data sharing for bootstrap classes is enabled by default</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">Class data sharing is enabled by default for bootstrap classes, unless your application is running in a container. You can use the <tt>-Xshareclasses</tt> option to change the default behavior.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/6541">#6541</a></td>
+<td valign="top">New option to share anonymous classes in the shared classes cache</td>
```
Comment (DanHeidinga) [inline doc/release-notes/0.16/0.16.md]: `share anonymous classes` -> `share VM anonymous classes`

File: doc/release-notes/0.16/0.16.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">Anonymous classes are now stored in the shared classes cache by default so that they are available for ahead-of-time (AOT) compilation. To
```
Comment (DanHeidinga) [inline doc/release-notes/0.16/0.16.md]: `Anonymous classes are` -> `Anonymous classes, those defined by Unsafe.defineAnonymousClass, are`

File: doc/release-notes/0.16/0.16.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/5164">#5164</a></td>
+<td valign="top">New <tt>jcmd</tt> tool</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">For compatibility, OpenJ9 includes an independent implementation of the <tt>jcmd</tt> tool, which can run diagnostic commands on a specified VM or VMs.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/3743">#3743</a></td>
+<td valign="top">Automatically setting an initial heap size by default</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">In release 0.15, a new feature was introduced that allowed OpenJ9 to learn and set an appropriate initial heap size for an application as an alternative to a user manually sizing and setting an <tt>-Xms</tt> value. This feature could be enabled by setting the <tt>-XX:+useGCStartupHints</tt> option on the command line. This
```
Comment (DanHeidinga) [inline doc/release-notes/0.16/0.16.md]: `-XX:+useGCStartupHints`  Does the option start with a lower case `u` or is this a typo?  We typically ensure they start with a capital

Comment (pshipton) [reply doc/release-notes/0.16/0.16.md]: The option is `-XX:+UseGCStartupHints`

Comment (SueChaplain) [reply doc/release-notes/0.16/0.16.md]: Fixed.

File: doc/release-notes/0.16/0.16.md (hunk 1 above)
Comment (SueChaplain) [reply doc/release-notes/0.16/0.16.md]: Fixed.

File: doc/release-notes/0.16/0.16.md (hunk 2 above)
Comment (SueChaplain) [reply doc/release-notes/0.16/0.16.md]: Fixed

**Comment by SueChaplain** [comment]
Added a new row to the changes table for #6536 / #5480 - shared cache generation number.
Think this is now good for MS1

**Pull request merged**